test/python/scripts/test_st_appconfig.py

```diff
@@ -38,8 +38,10 @@ def captured_output():
 
 class TestAppconfig(unittest.TestCase):
 
-    instance = "zen-edge-sample-icp1-blitz-env"
+    instance = os.environ['STREAMS_INSTANCE_ID']
```

Comment:
These fields should be instance fields, not class fields. Set them in a `setUp` method.